Samples/statementOfHoldings.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+/** @noinspection PhpUnhandledExceptionInspection */
+
+/**
+ * SAMPLE - Displays the statement of account for a specific depot.
+ */
+
+// See login.php, it returns a FinTs instance that is already logged in.
+/** @var \Fhp\FinTs $fints */
+$fints = require_once 'login.php';
+
+// Just pick the first account, for demonstration purposes. You could also have the user choose, or have SEPAAccount
+// hard-coded and not call getSEPAAccounts() at all.
+$getSepaAccounts = \Fhp\Action\GetSEPAAccounts::create();
+$fints->execute($getSepaAccounts);
+if ($getSepaAccounts->needsTan()) {
+    handleStrongAuthentication($getSepaAccounts); // See login.php for the implementation.
+}
+$oneAccount = $getSepaAccounts->getAccounts()[0];
+
+
+$getStatement = \Fhp\Action\GetDepot::create($oneAccount);
+$fints->execute($getStatement);
+if ($getStatement->needsTan()) {
+    handleStrongAuthentication($getStatement); // See login.php for the implementation.
+}
+
+$soa = $getStatement->getStatement();
+foreach ($soa->getStatements() as $statement) {
+			echo '=======================================' . PHP_EOL;
```

Comment:
All of the tab characters should be 4 spaces. `composer run cs-fix` should fix that (and ideally GitHub's checks should automatically detect that).